### ISSUE TITLE
Some questions about the supplementary materials and the training details

### ISSUE THREAD

**Issue by mmSir** [sniklaus/softmax-splatting#30]
Here are two questions:

1. Where can i get the supplementary materials? I haven't found it in the paper or your website...
2. The second question is about the training details: It was mentioned in the paper, you trained portion of the Vimeo-90k dataset. I would like to know the approximate amount of data. Is 5000 triplets enough? Training is too slow on my poor GPU.

Looking forward to your reply!~

**Comment by sniklaus** [comment]
You can find the supplementary [here](https://openaccess.thecvf.com/content_CVPR_2020/html/Niklaus_Softmax_Splatting_for_Video_Frame_Interpolation_CVPR_2020_paper.html). Reducing the number of samples to a mere 5000 is probably not a good idea. It would probably be better to just train for fever iterations in your case (why train on fewer samples if you can just do fewer iterations/epochs).

**Comment by mmSir** [comment]
Thanks for your reply!
